virtcontainers/cgroups.go

```diff
@@ -192,28 +215,28 @@ func (s *Sandbox) deleteCgroups() error {
 	}
 
 	// move running process here, that way cgroup can be removed
-	parent, err := parentCgroup(V1NoConstraints, path)
+	parent, err := parentCgroup(cgroupTypes, path)
 	if err != nil {
 		// parent cgroup doesn't exist, that means there are no process running
 		// and the no constraints cgroup was removed.
 		s.Logger().WithError(err).Warn("Parent cgroup doesn't exist")
 		return nil
 	}
 
-	if err := noConstraintsCgroup.MoveTo(parent); err != nil {
+	if err := cgroup.MoveTo(parent); err != nil {
 		// Don't fail, cgroup can be deleted
 		s.Logger().WithError(err).Warn("Could not move process from no constraints to parent cgroup")
 	}
-
-	return noConstraintsCgroup.Delete()
+	return cgroup.Delete()
 }
 
-func (s *Sandbox) constrainHypervisor(cgroup cgroups.Cgroup) error {
+func (s *Sandbox) constrainHypervisorVCPUs(cgroup cgroups.Cgroup) error {
```

Comment:
I understand that we are clarifying what the function does by updating the name of it, however, this probably should be in a separate commit, since it isn't really about the cgroup change, right?  Also, /me wonders if we should be more diligent about adding comments for functions, even if they aren't exported.